include/configs/imx8mm_var_dart.h

```diff
@@ -89,8 +89,10 @@
 	"get_fit_config=leica_mcu_comm board_id board_id;" \
 		"if test \"${board_id}\" < \"D\"; then" \
 		"    env set fit_config conf-freescale_ap20-pt1.dtb;" \
-		"else" \
+		"elif test \"${board_id}\" = \"D\"; then" \
 		"    env set fit_config conf-freescale_ap20-pt1-5.dtb;" \
+		"else" \
+		"    env set fit_config conf-freescale_ap20.dtb;" \
```

Comment:
This assumption looks very strong. Please provide more info in commit message: is there really a difference between `ap20` and `ap20-pt1.5`?
My opinion, they are the same and we need just replace `conf-freescale_ap20-pt1-5.dtb` line with `conf-freescale_ap20.dtb`

Reply:
keep pt1 and pt1-5. create a new ap20, which is a copy pt1-5
load pt1 for revision below D, load pt1-5 for revision D, load ap20 for revision above D.

Reply:
@lidaide indeed, we have a problem here, i didn't think that through enough, sorry
this conflicts with removing the dtb files from the geosurv....

My recommendation would be to fully drop support for pt1 and pt1.5:

```
"if test \"${board_id}\" < \"D\"; then" \
		"    DISPLAY ERROR MESSAGE AND NO BOOT" \
		"else" \
		"    env set fit_config conf-freescale_ap20.dtb;" \
```
What do you think? @grembeter